logstash-core/lib/logstash/logging/slow_logger/top_items.rb

```diff
@@ -0,0 +1,69 @@
+# encoding: utf-8
+require "logstash/namespace"
+require "set"
+require 'thread'
+
+module LogStash; module Logging; module Util
+
+  class Operation
```

Comment:
Did you intend for `Operation` to be an "inner" class of `TopItems`? If so it needs to be indented inside `TopItems`